distribution/apim-accelerator/resources/repository/conf/deployment.toml

```diff
@@ -418,6 +418,13 @@ policy.limit.time_unit = "min"
 patient_id_uri = "http://wso2.org/claims/patientId"
 patient_id_key = "patientId"
 
+[healthcare.backend.auth]
```

Comment:
this should be a toml array right? in order to support multiple backends

Reply:
fixed in [b8da50d](https://github.com/wso2/healthcare-accelerator/pull/7/commits/b8da50dbe832d38ee8259489c5c03798ba7ca81d)